versioned_docs/version-1.3/guides/setting-up-development-project.md

```diff
@@ -13,7 +13,6 @@ keywords: [get-started, local-install]
 - [Desktop Modeler](https://camunda.com/download/modeler/)
 - [Operate](/self-managed/operate-deployment/install-and-start.md)
 - [Tasklist](/self-managed/tasklist-deployment/install-and-start.md)
-- [Optimize](../components/optimize/what-is-optimize.md)
```

Comment:
wooooops. I think I know how this got removed. I originally moved the first doc over (what-is-optimize) a while ago, in a local branch; I bet to get past broken markdown links back then I just deleted these references.

I'll put them back.